nss-day-cohort-41/tabloid-fullstack-elvis_spotters
# Aj category view all

Description:
Added View All Categories feature.
Connected Category to the Admin Tools Dropdown
Fixed #9 
Testing instructions:
Open the application and log in
You should be directed to posts
Navigate to the admin tool dropdown
click Category Manager
You should be presented with a list of categories in order with options to edit and delete

[x] Adding Feature to Tabloid
[x] Provided Testing Instructions
[x] No Breaking Changes.
[x] Everything is withing style and code guidelines
